api/DHSC.FingertipsNext.Modules.Area.Repository/AreaRepository.cs

```diff
@@ -86,25 +73,25 @@ string childAreaType
 
        var area = await _dbContext
             .Area
-            .Include(a => a.AreaType)
+            .Include(area => area.AreaType)
             .Include(a => a.Children)
-                .ThenInclude(a => a.Children)
+                .ThenInclude(area => area.Children)
 
-                .Include(a => a.Children)
-                .ThenInclude(x => x.Parents)
+                .Include(area => area.Children)
+                .ThenInclude(area => area.Parents)
 
-                .Include(a => a.Children)
-                .ThenInclude(x => x.AreaType)
+                .Include(area => area.Children)
+                .ThenInclude(area => area.AreaType)
```

Comment:
I think this makes things a little less readable here.  Could we replace x with a different name to 'area' to distinguish that its working on either the child or parent.